src/viam/sdk/components/camera/camera.hpp

```diff
@@ -3,8 +3,10 @@
 /// @brief Defines a `Camera` component.
 #pragma once
 
+#include <arpa/inet.h>
 #include <bitset>
 #include <chrono>
+#include <cstring>
```

Comment:
Do we actually need these new includes? I don't see where they're being used here.

In particular the `arpa/inet.h` include is something I'd like to avoid if possible, since it isn't part of the standard library and so creates an additional dependency.

Note it's possible I'm being overly paranoid here! I'm not sure how standard the `arpa/inet.h` include is, so happy to have some pushback on this :)